use-cases/wordpress-k8s-mysql/README.md

```diff
@@ -94,28 +95,28 @@ Customize the values if needed.
 
 ```bash
 cd 01-kube
-terraform init
-terraform plan -var-file=../variables_01.tfvars
+tofu init
```

Comment:
do not replace terraform CLI by opentofu CLI but instead add a note telling the users that they can use opentofu CLI if they want.